docs/index.md

```diff
@@ -1,9 +1,12 @@
 ---
 lang: en-us
 title: Home
-description: Home for the Pulsar Editor
-sidebar: false
 home: true
+actions:
+  - text: "Get Started 💡"
+    link: "/docs/launch-manual"
+heroText: Pulsar
+features:
+  - title: Teletype for Pulsar
+    details: "Great things happen when developers work together—from teaching and sharing knowledge to building better software. Teletype for Pulsar makes collaborating on code just as easy as it is to code alone, right from your editor. Share your workspace and edit code together in real time. To start collaborating, open Teletype in Pulsar and install the package."
```

Comment:
Still not convinced we should be displaying this on the live site. If we need a placeholder just do a `lorum ipsum`